Topic Wise Questions with Solutions/Array/Hourglass_Sum.py

```diff
@@ -0,0 +1,65 @@
+'''
+Given a 6X6 2D Array, A:
+1 1 1 0 0 0
+0 1 0 0 0 0
+1 1 1 0 0 0
+0 0 0 0 0 0
+0 0 0 0 0 0
+0 0 0 0 0 0
+
+An hourglass in A is a subset of values with indices falling in this pattern 
+in A's graphical representation:
+a b c
+d

```

Comment:
it should be like this
a b c
&ensp; d
e f g

Reply:
Done @rcoder23, please check now.